src/main/java/org/isf/visits/rest/VisitsController.java

```diff
@@ -96,7 +96,7 @@ public ResponseEntity<List<VisitDTO>> getVisit(@PathVariable("patID") int patID)
 	 * @return an error if there are some problem, the visitor id (Integer) otherwise
 	 * @throws OHServiceException
 	 */
-	@PostMapping(value = "/visits", produces = MediaType.APPLICATION_JSON_VALUE)
+	@PostMapping(value = "/visit", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Didn't a recent update make this plural?  why change it back?

Reply:
For two reasons:
1) It is not possible to have two @PostMapping with the same value
2) In this method we want to create a single visit, on the other one (the one mapped on "/visits") we want to create a list of visit, so for coherence this method shoud be mapped on "/visit"

Reply:
I see, I didn't realize that! Nevertheless, I think this could be error prone... could we change it to something totally different, like `POST /visits/list`?

(FWIW and IMHO, I prefer singular endpoints for a number of reasons (plurals may differ (visit-s, laborator-ies, ecc...) and make everything more verbose, but I decided to follow the standard we have so far in the repo)

Reply:
I've reused the `insertList` pattern, already used in oh-api for Laboratories;
please, check last commit.

Reply:
yes, it's ok for me then